okta-hosted-login/src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+micronaut:
+  security:
+    authentication: idtoken 
+    oauth2:
+      clients:
+        okta:
+          client-id: '${OKTA_OAUTH2_CLIENTID}'
+          client-secret: '${OKTA_OAUTH2_CLIENTSECRET}'
+          openid:
+            issuer: '${OKTA_OAUTH2_ISSUER}'
+    endpoints:
+      logout:
+        path: '/oauth/logout'
```

Comment:
I believe you had `/logout` configured earlier when you registered this app in Okta. The values should match.

Reply:
yes, added a comment.